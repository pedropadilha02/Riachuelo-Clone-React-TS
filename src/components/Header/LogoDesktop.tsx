import styled from 'styled-components';

const Logo = styled.h1`
  height: 60px;
  line-height: 60px;
  text-align: center;
  margin: 0;

  @media (max-width: 1200px) {
    display: none;
  }
`;

export function LogoDesktop() {
  return (
    <Logo>
      <img src="http://localhost:3000/assets/riachuelo.svg" alt="" />
    </Logo>
  );
}
