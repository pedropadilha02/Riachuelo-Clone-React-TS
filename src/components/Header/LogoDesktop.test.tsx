import { render } from '@testing-library/react';
import { LogoDesktop } from './LogoDesktop';

describe('Teste de renderização do LogoDesktop', () => {
  it('O snapshot do component deve permanecer semre o mesmo', () => {
    const { container } = render(<LogoDesktop />);

    expect(container).toMatchSnapshot();
  });
});
