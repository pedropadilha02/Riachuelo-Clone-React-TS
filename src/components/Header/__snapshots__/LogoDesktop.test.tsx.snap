// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Teste de renderização do LogoDesktop O snapshot do component deve permanecer semre o mesmo 1`] = `
<div>
  <h1
    class="sc-bdnxRM gpQfJq"
  >
    <img
      alt=""
      src="http://localhost:3000/assets/riachuelo.svg"
    />
  </h1>
</div>
`;
